terminal_api_flows/__init__.py

```diff
@@ -65,6 +65,36 @@ def skip_screen():
     print(f"    `-------------payload: {json_data['status']}")
 
 
+def _toggle_screensaver(screensaver: bool) -> None:
+    res, json_data = http_request(
+        endpoint="actions",
+        action="POST",
+        json_body=json.dumps(
+            {
+                "action": "toggle_screensaver",
+                "data": {
+                    "screensaver_enabled": screensaver
+                }
+            }
+        )
+    )
+
+    if res.status == 200 and json_data["status"] == "SUCCESS":
+        print_outcome("SUCCESS", res.status, json_data)
+    else:
+        print_outcome("FAILED", res.status, json_data)
+
+
+def turn_on_screensaver() -> None:
+    print("Turning ON screensaver:")
+    _toggle_screensaver(screensaver=True)
+
+
+def turn_off_screensaver() -> None:
+    print("Turning OFF screensaver:")
+    _toggle_screensaver(screensaver=False)
```

Comment:
Let's change the wording of this to: `Deactivating screensaver on customer terminal`

Reply:
Fixed it in cb6f630 :heavy_check_mark: